src/displayapp/screens/WatchFaceInfineat.h

```diff
@@ -0,0 +1,144 @@
+#pragma once
+
+#include <lvgl/src/lv_core/lv_obj.h>
```

Comment:
I think we're using lvgl.h directly, instead of including from the internals of the lvgl library

```suggestion
#include <lvgl/lvgl.h>
```

Reply:
Done